fs/fs.go

```diff
@@ -456,19 +473,78 @@ func (self *RealFsInfo) GetDirUsage(dir string, timeout time.Duration) (uint64,
 	err = cmd.Wait()
 	timer.Stop()
 	if err != nil {
-		return 0, fmt.Errorf("du command failed on %s with output stdout: %s, stderr: %s - %v", dir, string(stdoutb), string(stderrb), err)
+		return 0, fmt.Errorf("cmd %v failed with output stdout: %s, stderr: %s - %v", cmd.Args, string(stdoutb), string(stderrb), err)
 	}
 	stdout := string(stdoutb)
 	if souterr != nil {
 		glog.Errorf("failed to read from stdout for cmd %v - %v", cmd.Args, souterr)
 	}
 	usageInKb, err := strconv.ParseUint(strings.Fields(stdout)[0], 10, 64)
 	if err != nil {
-		return 0, fmt.Errorf("cannot parse 'du' output %s - %s", stdout, err)
+		return 0, fmt.Errorf("cannot parse cmd %v output %s - %s", cmd.Args, stdout, err)
 	}
 	return usageInKb * 1024, nil
 }
 
+func (self *RealFsInfo) GetDirInodeUsage(dir string, timeout time.Duration) (uint64, error) {
+	if dir == "" {
+		return 0, fmt.Errorf("invalid directory")
+	}
+	claimFindToken()
+	defer releaseFindToken()
+	findCmd := exec.Command("find", dir, "-xdev")
```

Comment:
I think this will double count inodes on base layers shared across containers. That's probably what we want, but it means killing a countainer will not reclaim all it's reported inodes.